app/renderer/js/utils/default-util.js

```diff
@@ -1,9 +1,22 @@
 const fs = require('fs');
-const { app } = require('electron').remote;
 
-const logDir = `${app.getPath('userData')}/Logs/`;
+let app = null;
+if (process.type === 'renderer') {
+	app = require('electron').remote.app;
+} else {
+	app = require('electron').app;
+}
 
+const zulipDir = app.getPath('userData');
+const logDir = `${zulipDir}/Logs/`;
 const initSetUp = () => {
+	// if it is the first time the app is running
+	// create zulip dir in userData folder to
+	// avoid errors
+	if (!fs.existsSync(zulipDir)) {
```

Comment:
The one thing that worries me here is that the app directory is ideally created by the electron. I hope this doesn't get messed up with that thing 😕

Reply:
I confirmed that it will add all the files are added afterwards.

Reply:
@cPhost so we don't need to worry, right?